contrib/opencensus-ext-flask/opencensus/ext/flask/flask_middleware.py

```diff
@@ -105,90 +87,28 @@ def init_app(self, app):
         # get settings from app config
         settings = self.app.config.get('OPENCENSUS_TRACE', {})
 
-        self.sampler = (self.sampler
-                        or settings.get('SAMPLER',
-                                        self.DEFAULT_SAMPLER))
-        self.exporter = (self.exporter
-                         or settings.get('EXPORTER',
-                                         self.DEFAULT_EXPORTER))
-        self.propagator = (self.propagator
-                           or settings.get('PROPAGATOR',
-                                           self.DEFAULT_PROPAGATOR))
-
-        # get params from app config
-        params = self.app.config.get('OPENCENSUS_TRACE_PARAMS', {})
```

Comment:
Yep.
And probably instead of having `OPENCENSUS_TRACE`, use `OPENCENUS` as the configuration name (and having the second level `TRACE`, `METRICS`, etc.).